### PR TITLE
OKTA-488909 :- rolling back fix for bug OKTA-476273

### PR DESCRIPTION
**Pull request by vicentevillegas-okta**
## Description:
This PR will rollback the fix on [OKTA-476273](https://oktainc.atlassian.net/browse/OKTA-476273). We need to figure out a better way to fix the original problem. The rollback should be enough for now because the original bug is more a "cosmetic" issue.

I wanted to include a testcafe test and make sure we won't have the same issue in the future, however there is a limitation (bug) in testcafe, you can see more details [here](https://github.com/DevExpress/testcafe/issues/1186).
What we are going to do to develop a selenium test to validate this case for MFA in the classic version. I do believe selenium is robust enough and 🤞 it won't have the same limitation than testcafe. This is the [Jira for the selenium test](https://oktainc.atlassian.net/browse/OKTA-505855)

## PR Checklist

- [x] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

- Did you verify the change by running downstream monolith artifacts? (YES | NO | UNSURE)
   YES, you can see [bacon here](https://bacon-go.aue1e.saasure.net/commits?artifact=monolith&branch=VV-OKTA-488909-SIW-testing&page=1&pageSize=1)

### Screenshot/Video:
aa

### Reviewers:
@justinabrokwah-okta 

@okta/ciamx 

### Issue:
- [OKTA-488909](https://oktainc.atlassian.net/browse/OKTA-488909)

### Bacon
https://bacon-go.aue1e.saasure.net/commits?artifact=okta-signin-widget&branch=VV-OKTA-488909&page=1&pageSize=1

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "private": true,
   "name": "@okta/okta-signin-widget",
   "description": "The Okta Sign-In Widget",
-  "version": "6.4.1",
+  "version": "6.4.2",
```
Comment (shuowu-okta) [inline package.json:5]: Can you revert this change? widget-coordinator will handle version bump during release

Comment (vicentevillegas-okta) [reply package.json:5]: sure, thank you for let me know!

**Review by justinabrokwah-okta — COMMENTED** [review]
LGTM pending removal of the version change in `package.json`. If the okta-core bacon run is green, you should post it here as well

**Review by justinabrokwah-okta — APPROVED** [review]

**Review by shuowu-okta — APPROVED** [review]
